packages/browser/src/__tests__/autocapture-utils.test.ts

```diff
@@ -398,34 +398,36 @@ describe(`Autocapture utility functions`, () => {
             expect(shouldCaptureElement(input)).toBe(false)
         })
 
-        it(`should not include fields with sensitive names`, () => {
-            const sensitiveNames = [
-                `cc_name`,
-                `card-num`,
-                `ccnum`,
-                `credit-card_number`,
-                `credit_card[number]`,
-                `csc num`,
-                `CVC`,
-                `Expiration`,
-                `password`,
-                `pwd`,
-                `routing`,
-                `routing-number`,
-                `security code`,
-                `seccode`,
-                `security number`,
-                `social sec`,
-                `SsN`,
-            ]
-            sensitiveNames.forEach((name) => {
-                input.name = ''
-                expect(shouldCaptureElement(input)).toBe(true)
+        it.each([
+            'cc_name',
+            'card-num',
+            'ccnum',
+            'credit-card_number',
+            'credit_card[number]',
+            'csc num',
+            'CVC',
+            'Expiration',
+            'password',
+            'pwd',
+            'routing',
+            'routing-number',
+            'security code',
+            'seccode',
+            'security number',
+            'social sec',
+            'SsN',
+        ])(`should not include fields with sensitive name: %s`, (name) => {
+            input.name = name
+            expect(shouldCaptureElement(input)).toBe(false)
+        })
 
+        it.each(['document-expiry', 'success_message', 'account-settings', 'compass-icon'])(
```

Comment:
`document-expiry` and `compass-icon` would have failed before because of `exp` and `pass` appearing anywhere in the string.

`success_message` and `account-setting` would have passed because `cc` wasn't at the start.